hawkular-apm-server/standalone-wrapper.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# where are we located? usually, we are on $JBOSS_HOME/bin , along with the other scripts
+DIRNAME=`dirname "$0"`
+
+# we create only if there's no active users starting with 'admin'
+grep "^admin" "${DIRNAME}/../standalone/configuration/application-users.properties" > /dev/null
+RT=$?
+
+if [ $RT -eq 0 ]; then
+  echo "An admin user has already been found."
+elif [ $RT -ne 1 ]; then
+  echo "An error has been found when attempting to check if an admin user exists. Aborting."
+  exit 1
+else
+  username="admin$(head /dev/random -c 512 | tr -dc A-Z-a-z-0-9 | head -c 7)"
```

Comment:
The problem with env vars is that they are very insecure. This way, the password is shown only once, whereas with an env var, the password would likely be passed for all runs of this container (and thus, available in plaintext via `docker inspect`). There are some new features in Docker that I need to catch up, like volumes. Perhaps that could be a solution for a production quality image.